### ISSUE TITLE
library is not types (py.typed)

### ISSUE THREAD

**Issue by ssbarnea** [spdx/tools-python#500]
I observed that this library is not typed yet as mypy reports:
```
error: Skipping analyzing "spdx.config": module is installed, but missing library stubs or py.typed marker  [import]
note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

While adding types can take some time this can also be done gradually, starting with adding a `py.typed` marker file being a first step.

**Comment by maxhbr** [comment]
Yes, we are aware of that. We do not plan to fix that for 0.7.x, but it is in the backlog for the currently happening rewrite.

**Comment by ssbarnea** [comment]
Thanks for confirmation, adding an exception was easy anyway. I am very glad to hear that you plan to address it in the future.

**Comment by armintaenzertng** [comment]
We added type hints and a `py.typed` file in the refactored version that can now be found on main.

**Comment by ssbarnea** [comment]
@armintaenzertng Apparently even with the latest release, 0.7.1 I got some type errors reported by mypy, so maybe we should reopen the bug.

```
test/test_schemas.py:13: error: Skipping analyzing "spdx.config": module is installed, but missing library stubs or py.typed marker  [import]
test/test_schemas.py:13: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
test/test_schemas.py:13: error: Skipping analyzing "spdx": module is installed, but missing library stubs or py.typed marker  [import]
```

**Comment by armintaenzertng** [comment]
Hi @ssbarnea, thanks for your continued interest in this topic! :)
As stated above, we do not plan to include typing in the 0.7.x versions but have it fully implemented in the upcoming 0.8.x version. If you like, please try out the current alpha releases that are [available on PyPI](https://pypi.org/project/spdx-tools/0.8.0a2/#history) or the current state of the `main` branch right here on this repo.
